discovery-provider/src/trending_strategies/ePWJD_trending_tracks_strategy.py

```diff
@@ -47,5 +55,102 @@ def __init__(self):
     def get_track_score(self, time, track):
         return z(time, track)
 
+    def update_track_score_query(self, session):
+        start_time = time.time()
+        trending_track_query = text(
+            """
+            begin;
+                DELETE FROM track_trending_scores WHERE type=:type AND version=:version;
+                INSERT INTO track_trending_scores 
```

Comment:
Feels like a bummer to repeat the SELECT 3x but I can't think of a better (and also more straightforward) way to do it.